http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -211,18 +240,6 @@ class ProxyService(
       r <- proxyInternal(hostContext, request)
     } yield r
 
-    res.onError {
```

Comment:
I think this was causing duplicate (noisy) logging. I don't think `equest.entity.discardBytes()` is necessary either; we had added that to address some apache issues but it turned out to be something else.

Reply:
there're a few places where we're doing `onError`, but then also `handleErrorWith` at a later point...would be good to address these slowly